fearless/Modules/WalletMainContainer/WalletMainContainerPresenter.swift

```diff
@@ -159,3 +161,21 @@ extension WalletMainContainerPresenter: SelectNetworkDelegate {
         interactor.saveChainIdForFilter(chain?.chainId)
     }
 }
+
+extension WalletMainContainerPresenter: ScanQRModuleOutput {
```

Comment:
methods in this protools do the same. Maybe remove didFinishWith(addressInfo: AddressQRInfo)